src/components/atoms/Alert/Alert.tsx

```diff
@@ -81,13 +97,29 @@ const IconWrapper = styled.div<{ severity: Severity }>`
   }
 `;
 
-const Alert: FC<AlertProps> = ({ severity = 'info', inline = false, children, ...rest }) => {
-  const Icon = MAP_BY_SEVERITY[severity].component;
+const CrossIcon = styled(Icon)`
+  position: absolute;
+  cursor: pointer;
+  right: 10px;
+  top: 10px;
+`;
+
+const Alert: FC<AlertProps> = ({ severity = 'info', inline = false, onRequestClose, children, ...rest }) => {
+  const { component: Icon, text } = MAP_BY_SEVERITY[severity];
 
   return (
     <Wrapper severity={severity} inline={inline} {...rest}>
       <IconWrapper severity={severity}>
         <Icon />
+        {onRequestClose && (
+          <CrossIcon
+            onClick={onRequestClose}
+            color={text}
+            variant={faTimes}
+            data-testid="ZA.alert-cross-icon"
```

Comment:
we're sure we want to hardcode it?

Reply:
for now yes, just as the modal one. If we see we need more than one alert in a page we can tweak it later